blaine-t.md

```diff
@@ -0,0 +1,62 @@
+---
+name: "@blaine-t"
+project: "The Hotswappable Battery Bank"
+---
+
+# The Hotswappable Battery Bank
+
+## Summary
+
+I am planning to build a custom battery bank that fits my needs and I can use in college and beyond.
```

Comment:
hey @blaine-t :)) you mentioned here that you can "use in college and beyond", does this mean you're currently a high school senior (about to go to college) or are you in college?

Reply:
Yes I am a current high schooler, sorry for the weird wording. I want to use it now and when I'm in college.